scqubits/utils/spectrum_utils.py

```diff
@@ -396,7 +396,7 @@ def recast_esys_mapdata(
 def identity_wrap(
     operator: Union[str, ndarray, Qobj],
     subsystem: "QuantumSys",
-    subsys_list: List["QuantumSys"],
+    subsys_list: Tuple["QuantumSys"],
```

Comment:
If this is a tuple (rather than a list) of potentially multiple `QuantumSys` instances, then we need 
`Tuple["QuantumSys", ...]`